### ISSUE TITLE
Start using UI framework or libarary

### ISSUE THREAD

**Issue by l-margiela** [cgm616/0rc#19]
I think we should use React.js for the sake of flexibility. Introducing new features would be easer using some modern frontend framework (like Angular) or React.

**Comment by cgm616** [comment]
Good idea. I'm not well-versed in js, but we should use a Coffeescript framework if anything to make it compatible with the rest of ZeroNet and ZeroChat IRC in general.

**Comment by antiphase0** [comment]
I haven't used any of these frameworks personally, but ZeroMail and ZeroHello use a React-like micro framework called [Maquette](http://maquettejs.org/), and the developer of [ZeroTodos](http://127.0.0.1:43110/1B8LmXYHzMGZzcRWoidAQb5SmKSyfjN63f) has recommended a similar framework [vue.js](http://vuejs.org/).

**Comment by cgm616** [comment]
+1 on that, Maquette seems like a good choice. Using the same framework as ZeroNet and the official sites will also make users more comfortable.

**Comment by l-margiela** [comment]
+1, Marquette looks clean and simple

**Comment by AngeloR** [comment]
Marquette sounds look a good choice, especially if it's something that other ZeroNet devs would need to become familiar with.

**Comment by cgm616** [comment]
Being addressed at [xaxes/zeronet-relay-chat](http://github.com/xaxes/zeronet-relay-chat).
